doc/source/_templates/sidebar-nav-bs.html

```diff
@@ -0,0 +1,9 @@
+<nav class="bd-links" id="bd-docs-nav" aria-label="Main navigation">
+  <div class="bd-toc-item active">
+    {% if pagename.startswith("reference") %}
+    {{ generate_nav_html("sidebar", maxdepth=4, collapse=True, includehidden=True, titles_only=True) }}
+    {% else %}
+    {{ generate_nav_html("sidebar", maxdepth=4, collapse=True, includehidden=True, titles_only=True) }}
```

Comment:
These two lines look equivalent, did you mean to change one of them? If not, then the conditional can presumably be removed

Reply:
It may be useful to keep, just cause we may want to tweak it more and this syntax is very annoying to figure out.

I did keep it on purpose, the earlier commit (which matches what Pandas did) has `collapse=False` for the second line.

Reply:
I think if we play with `:orphan:` in `source/_templates/autosummary/function.rst` we _may_ want to go back to `collapse=False` for one of these, not sure.

Reply:
Okay, makes sense to keep it